examples/simple-lua/run.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Simple run script for the Lua example
+
+echo "=== SQLite-Vec Lua Example Runner ==="
+
+# Check if Lua is available
+if ! command -v lua &> /dev/null; then
+    echo "Error: Lua is not installed or not in PATH"
+    exit 1
+fi
+
+# Check if lsqlite3 is available
+lua -e "require('lsqlite3')" 2>/dev/null
+if [ $? -ne 0 ]; then
+    echo "Error: lsqlite3 module is not installed"
+    echo "Install it with: luarocks install lsqlite3"
```

Comment:
Maybe it would be better to print errors to stderr:
```suggestion
    echo "Error: lsqlite3 module is not installed" >&2
    echo "Help: luarocks install lsqlite3" >&2
```